Added a metadata table

Alopalao opened the pull request:
Base of Issue #13 
Added a metadata table
Added active, enabled, lldp, nni and uni items to display.
This pull request depends on changes to [k-info-panel/switch_info.kytos](https://github.com/kytos-ng/topology/blob/master/ui/k-info-panel/switch_info.kytos). Especifically:
From lines 18 to 19:
```
         <k-interface :interface_id="interface.id" :name="interface.name" :port_number="interface.port_number" :mac="interface.mac" 
                      :speed="interface.speed" :key="interface.name" v-for="interface in this.interfaces">
```
to Lines 18 to 20:
```
         <k-interface :interface_id="interface.id" :name="interface.name" :port_number="interface.port_number" :mac="interface.mac" 
                      :speed="interface.speed" :key="interface.name" :enabled="interface.enabled" :metadata="interface.metadata"
                      :active="interface.active" :lldp="interface.lldp" :nni="interface.nni" :uni="interface.uni" v-for="interface in this.interfaces">
```

viniarck reviewed (commented):
@Alopalao awesome contribution, looks good, but before we ship this let's make sure that:

1 - Please, open a PR on topology updating the `<k-interface>` components accordingly, we'll need to land both PRs. 
2 - Update the changelog file accordingly.
3 - Since this is the base `ui`, you'd also have to update the [`version` on `package.json`](https://github.com/kytos-ng/ui/blob/master/package.json#L4) just so we can release this as a beta version, and then developers or operators that want to try out the upcoming features they should be able to. You can set the version as `2022.2.0.b0`

Alopalao commented:
-The [PR](https://github.com/kytos-ng/topology/pull/85) on topology have been opened.
-Updated changelog.
-Updated version with suggested string. But my installed npm recognizes that version as an error, let me know if is a problem to change it to another version.

viniarck commented:
> -The [PR](https://github.com/kytos-ng/topology/pull/85) on topology have been opened. -Updated changelog. -Updated version with suggested string. But my installed npm recognizes that version as an error, let me know if is a problem to change it to another version.

@Alopalao

- I've pulled you're PR and successfully built with this version, which error did you end up with npm on your end? Example:

```
❯ npm run build

> kytos-web-ui@2022.2.0.b0 build
> npm run prod > /dev/null && zip -r latest.zip index.html dist

updating: index.html (deflated 50%)
updating: dist/ (stored 0%)
updating: dist/build.js.LICENSE.txt (deflated 59%)
updating: dist/icon-kytos.svg (deflated 57%)
updating: dist/build.js (deflated 71%)
updating: dist/logo-kytos.svg (deflated 59%)
```

- I forgot to mention that since we'll need to release `ui` as a beta version, the changelog changes should be under a new subsection, something like this, could you apply this patch?

```
diff --git a/CHANGELOG.rst b/CHANGELOG.rst
index a4d04ac..2aec413 100644
--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -7,8 +7,6 @@ UNRELEASED - Under development
 ******************************
 Added
 =====
--Added active, enabled, lldp, nni and uni items to ``kytos/interfaceInfo`` to be displayed in basic details
--Added a table to ``kytos/interfaceInfo`` to be displayed interface metadata

 Changed
 =======
@@ -28,6 +26,15 @@ Security
 Changed
 =======

+[2022.2.0.b0] - 2022-04-07
+**************************
+
+Added
+=====
+-Added active, enabled, lldp, nni and uni items to ``kytos/interfaceInfo`` to be displayed in basic details
+-Added a table to ``kytos/interfaceInfo`` to be displayed interface metadata
+
+
```

Alopalao commented:
Updated Changelog [d33cb83dbc58182ae1470ac8cedf5159b4d56b11](https://github.com/kytos-ng/ui/pull/23/commits/d33cb83dbc58182ae1470ac8cedf5159b4d56b11)

Alopalao commented:
About the version causing problems, it was because my npm and node were not updated from 6.14.4 and 10.19.0 respectively.

viniarck reviewed (approved):
Looks good to me, @Alopalao, thanks for this contributions and pushing the latest commits. Once this get merged and kytos-ng/topology PR https://github.com/kytos-ng/topology/pull/85/files also lands, and then I'll also bump `ui`'s tag version on GitHub just so `2022.2.0.b0` version is available for developers to try it out.
